### PR TITLE
Fix the root-cause of the edk_list proof issues

### PR DESCRIPTION
**Pull request by danielsn**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

# Check any applicable:
- [ ] Were any files moved? Moving files changes their URL, which breaks all hyperlinks to the files.

### CODE REVIEW

File: .cbmc-batch/jobs/aws_cryptosdk_edk_list_clean_up/aws_cryptosdk_edk_list_clean_up_harness.c [hunk 1]
```diff
@@ -35,9 +35,8 @@ void aws_cryptosdk_edk_list_clean_up_harness() {
     __CPROVER_assume(aws_cryptosdk_edk_list_elements_are_valid(&edk_list));
 
     aws_cryptosdk_edk_list_clean_up(&edk_list);
-    // assert(aws_cryptosdk_edk_list_is_valid(&edk_list));
-    // assert(aws_cryptosdk_edk_list_elements_are_valid(&edk_list));
+    assert(AWS_IS_ZEROED(edk_list));
 }
-
+#include <aws/common/error.inl>
```
Comment (feliperodri) [inline .cbmc-batch/jobs/aws_cryptosdk_edk_list_clean_up/aws_cryptosdk_edk_list_clean_up_harness.c]: Why do we need commented includes bellow this line? Why do we need to keep the include at the end of the proof harness (perhaps a comment here)?

File: .cbmc-batch/jobs/aws_cryptosdk_edk_list_clear/aws_cryptosdk_edk_list_clear_harness.c [hunk 2]
```diff
@@ -36,8 +36,8 @@ void aws_cryptosdk_edk_list_clear_harness() {
 
     aws_cryptosdk_edk_list_clear(&edk_list);
     assert(aws_cryptosdk_edk_list_is_valid(&edk_list));
-    assert(aws_cryptosdk_edk_list_elements_are_valid(&edk_list));
 }
 
+#include <aws/common/error.inl>
```
Comment (feliperodri) [inline .cbmc-batch/jobs/aws_cryptosdk_edk_list_clear/aws_cryptosdk_edk_list_clear_harness.c]: Why do we need commented includes bellow this line? Why do we need to keep the include at the end of the proof harness (perhaps a comment here)?

Comment (danielsn) [reply .cbmc-batch/jobs/aws_cryptosdk_edk_list_clear/aws_cryptosdk_edk_list_clear_harness.c]: Done

File: .cbmc-batch/jobs/aws_cryptosdk_edk_list_clean_up/aws_cryptosdk_edk_list_clean_up_harness.c (hunk 1 above)
Comment (danielsn) [reply .cbmc-batch/jobs/aws_cryptosdk_edk_list_clean_up/aws_cryptosdk_edk_list_clean_up_harness.c]: done

**Pull request merged**